Cargo.toml

```diff
@@ -11,6 +11,7 @@ publish = false
 [workspace]
 
 members = [
+    "balance",
```

Comment:
should this directory be named `tower-h2-balance`?

Reply:
All the `tower` subcrates are named `tower-*`, so I think probably.